packages/iTwinUI-react/src/core/ComboBox/ComboBox.tsx

```diff
@@ -70,8 +70,10 @@ export type ComboBoxProps<T> = {
   emptyStateMessage?: string;
   /**
    * A custom item renderer can be specified to control the rendering.
-   * This function should ideally return a customized version of `ComboBox.MenuItem`,
-   * otherwise you will need to make sure to provide styling for the `isFocused` state.
+   *
+   * For keyboard navigation to work, the returned element should use the `id` provided by this function.
+   * The `isFocused` state is calculated using this `id` and can be used for specifying the focus styling.
+   * If a `MenuItem` is returned, then focus sytling is automatically handled.
```

Comment:
`sytling` you say.
Where is vscode extension of spell checker????

Reply:
somehow missed it
![image](https://user-images.githubusercontent.com/9084735/168081333-fc0524cf-7c90-43e5-aedd-f2483f63281d.png)

fixed

Reply:
so bad marking... :O